### PR TITLE
Add colors to analysis diffs

### PR DESCRIPTION
**Pull request by Schaeff**

### CODE REVIEW

File: riscv/runtime/src/lib.rs [hunk 1]
```diff
@@ -45,9 +45,8 @@ extern "Rust" {
 }
 #[no_mangle]
 #[start]
-pub unsafe extern "C" fn __runtime_start() -> ! {
+pub unsafe extern "C" fn __runtime_start() {
     unsafe {
         main();
     }
-    loop {}
```
Comment (leonardoalt) [inline riscv/runtime/src/lib.rs:52]: Why is the loop not needed anymore?

File: riscv/src/compiler.rs [hunk 2]
```diff
@@ -189,6 +189,23 @@ impl asm_utils::compiler::Compiler for Risc {
         // for compilation, and will not be called.
         replace_coprocessor_stubs(&mut statements);
 
+        // Replace the tail call by a normal call followed by `return`. Quite hacky, should be done in the runtime instead.
```
Comment (leonardoalt) [inline riscv/src/compiler.rs:192]: Why not just put it there?

**Review by leonardoalt — COMMENTED** [review]
Not sure this PR was meant to be pushed like this? It has a lot more changes than just adding colors to diffs

**Comment by Schaeff** [comment]
need to rebase

File: riscv/runtime/src/lib.rs (hunk 1 above)
Comment (Schaeff) [reply riscv/runtime/src/lib.rs:52]: let's discuss on #392

Comment (leonardoalt) [reply riscv/runtime/src/lib.rs:52]: sure, but shouldn't this PR be just the cli file?

Comment (Schaeff) [reply riscv/runtime/src/lib.rs:52]: yes, need to rebase

Comment (leonardoalt) [reply riscv/runtime/src/lib.rs]: ahhh just noticed that this wasn't targeting `main`, ok will ignore for now then

**Review by leonardoalt — APPROVED** [review]

**Pull request merged**